shared/rakefiles/entrypoint.rake

```diff
@@ -101,7 +101,10 @@ task :display_cluster_info do
   puts "Flowmanager endpoint:"
   puts "  curl -k https://flowmanager.#{ENV["TF_VAR_domain_name"] }"
   puts
-  puts "Run `rake destroy` to delete all the expensive resources created by the deployment"
+  puts "Run `rake test_preferences` to execute Locust tests for Preferences."
```

Comment:
Why is this included here? It seems to be totally unrelated change to changing regions.

Reply:
It is unrelated. But it is just a minor comment update, which is here, because I noticed it while working on resource exhaustion issues. Would you prefer if I open separate GPII ticket and PR for this change?